src/AssemblerOptions.cpp

```diff
@@ -753,7 +761,30 @@ void AssemblerOptions::MarkerGraphOptions::parseSimplifyMaxLength()
 
 }
 
+void AssemblerOptions::ReadsOptions::parseDesiredCoverageString() {
+    desiredCoverage = std::stoull(desiredCoverageString);
 
+    size_t pos = desiredCoverageString.find_last_of("0123456789");
+    if (pos == desiredCoverageString.size() - 1) {
+        // Raw number of bases specified.
+        return;
+    }
+    pos++;
+    
+    while (isspace(desiredCoverageString[pos])) {
```

Comment:
We should not allow white space between the number and the multiplier.

Reply:
Agreed. For this to work, the parameter itself would need to be wrapped in `"`, making it a very unlikely scenario.